pyuvdata/uvcal/uvcal.py

```diff
@@ -1390,7 +1394,7 @@ def get_gains(self, ant, jpol=None, squeeze_pol=True):
         Returns
         -------
         complex ndarray
-            Gain solution of shape (Nfreqs, Ntimes, Npol) or (Nfreqs, Ntimes)
+            Gain solution of shape (Nfreqs, Ntimes, NJones) or (Nfreqs, Ntimes)
```

Comment:
Another nit-pick, but I think the parameter is `Njones` (i.e., lower case `j`). I think we should be consistent in all the docstrings.

Reply:
good catch! I found a few more and fixed them all.